pkg/agent/baker_test.go

```diff
@@ -1173,13 +1173,8 @@ func backfillCustomData(folder, customData string) {
 }
 
 func getDecodedVarsFromCseCmd(data []byte) (map[string]string, error) {
-	cseRegex, err := regexp.Compile(cseRegexString)
-	if err != nil {
-		return nil, fmt.Errorf("failed to compile regex: %s", err)
-	}
-
+	cseRegex := regexp.MustCompile(cseRegexString)
```

Comment:
why? this func returned the error already?

Reply:
Compile needed to change to MustCompile for the linter, and MustCompile handles the error

Reply:
what's the lint error? i'm confused why lint dislike that

maybe we should tweak some rules

Reply:
it is from gocritic 

```
pkg/agent/baker_test.go:1176:19: regexpMust: for const patterns like cseRegexString, use regexp.MustCompile (gocritic)
	cseRegex, err := regexp.Compile(cseRegexString)
```

Reply:
thanks!

I kinda disagree with this because I'm anti-panic, but super minor. we can tackle fixing things now, and fine tuning rules later.

Reply:
I guess the lint error is because known good constant regexes "can't" panic but I always feel "can't" is a strong word in the world of programming

Reply:
I would agree with that one haha